packages/block-library/src/separator/theme.scss

```diff
@@ -1,5 +1,6 @@
 .wp-block-separator {
 	border: none;
+	padding: 10px;
```

Comment:
On twenty nineteen theme the line is not vertically centered:

<img width="868" alt="screenshot 2018-12-18 at 09 57 00" src="https://user-images.githubusercontent.com/11271197/50146925-cab4fc80-02ac-11e9-81a7-49ad38930a0f.png">

Would it be possible to vertically center it?

Reply:
Depending how this is centered vertically — it _might_ be a 2019 specific issue not sure, haven't had time to test yet — we should use `$grid-unit` instead of `10px`.